contrib/js-1.5-rc6a+elinks.patch

```diff
@@ -1,61 +0,0 @@
-This quick'n'dirty patch enables relatively painless SpiderMonkey installation.
```

Comment:
Removed js-1.5-rc6a+elinks.patch because the Meson build script expect
SpiderMonkey 78 for which this patch is irrelevant. Also, you can build
elinks with the vanilla SpiderMonkey 78, that is without needing to
apply any patch.